lesson11/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="script.js" defer></script>
     <link rel="stylesheet" href="reset.css" type="text/css" />
     <link rel="stylesheet" href="style.css" type="text/css" />
     <title>TerraceTechフロントエンドエンジニア養成所</title>
```

Comment:
I change to read script with defer.
I add defer to the script and write it in the head, the page load speed will be faster.
This is because defer loads the script at the same time it parses the HTML.